packages/esm-ward-app/src/hooks/useInpatientRequest.ts

```diff
@@ -21,5 +21,6 @@ export function useInpatientRequest(dispositionType: Array<DispositionType> = ['
   return {
     inpatientRequests: data?.data?.results,
     ...rest,
+    error: null,
```

Comment:
Is this intentional? `rest` already has an error field.

Reply:
Thanks for the catch @chibongho, I was testing some cases.